admin/user_stories.txt

```diff
@@ -1,7 +1,10 @@
 #### As a student,
 
 As a student, I will be able to log into my profile
+As a student, I will be able to update my personal information in my profile
 As a student, I will be able to view my 341 qr code
+As a student, I will be able to receive notifications when a new 341 is filled out or updated.
+As a student, I will be able to request a review or appeal a discrepancy 341.
```

Comment:
I don't think this should be handled by the website.

Ultimately the MTL is going to talk to the student about the 341 (in person) which is where an appeal can be made